backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -158,7 +158,8 @@ def parent_data
   def parent
     if parent_data.present?
       @parent ||= parent_data[:model_class].
-          send("find_by_#{parent_data[:find_by]}", params["#{resource.model_name}_id"])
+        # Don't use find_by to workaround globalize/globalize#441 bug
+        send("find_by", parent_data[:find_by].to_s => params["#{resource.model_name}_id"])
```

Comment:
hi, @k3rn31 in your comment you specify that `find_by` should not be used but in the line below you use it anyway. Are you sure that this is ok? Besides that, could you please pass the first argument to `send` as a symbol?